src/components/Chip.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+interface chipProps {
+  active: boolean
+  icon: string | null
+  message: string
+}
+
+function Chip({ active, icon, message }: chipProps) {
```

Comment:
icon에 기본값을 null로 전달해주지 않아도 설정되게끔 아래처럼 전달하면 좋을 것 같습니다!
```suggestion
function Chip({ active, icon = null, message }: chipProps) {
```

Reply:
이게 defaultparameter설정하는거죵? 알겠습니다